src/ua/dp/ort/chapter_OOP_04/Stringch.java

```diff
@@ -0,0 +1,128 @@
+package ua.dp.ort.chapter_OOP_04;
+
+public class Stringch {
+	private Character[] _simbol; 
+	private int _length;
+	private final static int defaultLength = 30;
+		
+	/**
+	 * Создает пустую строку с длиной по умолчанию
+	 * 
+	 */
+	public Stringch() {
+		setDefaultArray(defaultLength);
+	}
+	
+	/**
+	 * Создает строку на заданное количество элементов со случайными значениями
+	 * @param length - количество элементов
+	 */
+	public Stringch(int length) {
+		setLengthArray(length);
+	}
+	
+	public void setLengthArray(int length) {
+		_simbol = new Character[length];
+		_length = length;
+		for (int i = 0; i < length; ++i) {
+			_simbol[i] = (char)((int)(Math.random() * 127) + 21);

```

Comment:
Why do we need random generating here? If you want to generate random content - do it in the public static void main method